src/scripts.js

```diff
@@ -10,32 +10,54 @@ console.log('This is the JavaScript entry file - your code begins here.');
 import { getData } from './apiCalls';
 import UserRepository from './UserRepository';
 import User from './User';
+import HydrationRepo from './HydrationRepo';
 import Hydration from './Hydration';
 
-let userRepository, user;
-// function to get data from the API
+let userRepository, user, hydrationRepo, hydration, userIndex;
+
+// function to create random number for user
+const getRandomIndex = () => Math.floor(Math.random() * 50);
+userIndex = getRandomIndex();
 
 const populateDataOnPage = () => {
   document.getElementById(
     'welcome'
   ).innerText = `Welcome ${user.getFirstName()}`;
   document.getElementById('step-goal').innerText = user.dailyStepGoal;
   document.getElementById('avg-step-goal').innerText =
-    userRepository.calculateAverageStepGoal();
+    userRepository.getAvgStepGoal();
   document.getElementById('name').innerText = user.name;
   document.getElementById('address').innerText = user.address;
   document.getElementById('email').innerText = user.email;
   document.getElementById('stride').innerText = user.strideLength;
+  // Try to refactor this and remove it from the dom manipulation
```

Comment:
This would go along with what Cassandra said about just having the call of that helper function here! Sounds good!